spec/suites/layer/vector/PolylineSpec.js

```diff
@@ -212,4 +212,15 @@ describe('Polyline', function () {
 			expect(polyline._latlngs).to.eql([L.latLng([1, 2])]);
 		});
 	});
+
+	describe("#style", function () {
+		it("can set weight after being added to the map", function () {
+			var polyline = new L.Polyline([]).addTo(map);
```

Comment:
In #6662 you were trying to use `setStyle` before adding to map.
But here you do it after adding.
Does it actually matter?